From 79efb02c7dbdd4e9dec4bfc20a2ed8d4875a34b3 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Fri, 9 Oct 2020 19:31:50 +0800
Subject: [PATCH] ARM: dts: rv1126-bat-ipc-v10: prefer use sc210iot for low
 power

Test scenario:
1080p rtsp (only video) preview
ov2718: 999.7mw
sc210iot: 793.5mw

The sc210iot is lower power dissipation about 206mw than ov2718.

Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
Change-Id: Ib743f9ca85ea911385f90620431e8592b0c70444
---
 arch/arm/boot/dts/rv1126-bat-ipc-v10.dts | 16 +++++++---------
 1 file changed, 7 insertions(+), 9 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
index 624a026ef20c..be0198bc4a10 100644
--- a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
+++ b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
@@ -197,25 +197,23 @@
 &i2c1 {
 	status = "okay";
 
-	ov2718: ov2718@10 {
-		compatible = "ovti,ov2718";
-		status = "okay";
-		reg = <0x10>;
+	sc210iot: sc210iot@32 {
+		compatible = "smartsens,sc210iot";
+		reg = <0x32>;
 		clocks = <&cru CLK_MIPICSI_OUT>;
 		clock-names = "xvclk";
 		power-domains = <&power RV1126_PD_VI>;
-		pinctrl-names = "rockchip,camera_default";
-		pinctrl-0 = <&mipicsi_clk0>;
 		avdd-supply = <&vcc2v8_avdd>;
 		dovdd-supply = <&vcc1v8_dovdd>;
 		dvdd-supply = <&vcc1v2_dvdd>;
-		pwd-gpios = <&gpio1 RK_PD4 GPIO_ACTIVE_HIGH>;
+		pwdn-gpios = <&gpio1 RK_PD4 GPIO_ACTIVE_HIGH>;
 		reset-gpios = <&gpio1 RK_PD5 GPIO_ACTIVE_LOW>;
 		rockchip,camera-module-index = <1>;
 		rockchip,camera-module-facing = "front";
-		rockchip,camera-module-name = "YT-RV1109-3-V1";
-		rockchip,camera-module-lens-name = "M43-4IR-2MP-F2";
+		rockchip,camera-module-name = "YT-SC210-V2";
+		rockchip,camera-module-lens-name = "M12-30IRC-2MP-F18";
 		ir-cut = <&cam_ircut0>;
+
 		port {
 			ucam_out0: endpoint {
 				remote-endpoint = <&mipi_in_ucam0>;
-- 
2.35.3

